examples/advance_music_cog.py

```diff
@@ -0,0 +1,457 @@
+import discord
+from discord.ext import commands
+
+import discordSuperUtils
+from discordSuperUtils import MusicManager, PageManager
+
+import datetime
+import time
+from typing import Optional
+
+#function to format duration
+def parse_duration(duration: Optional[float]) -> str:
+    return (
+        time.strftime("%H:%M:%S", time.gmtime(duration))
+        if duration != "LIVE"
+        else duration
+    )
+
+#format view, like and dislike count
+def parse_count(count: int):
```

Comment:
does not support numbers larger than 1 billion and logic is very bad

Reply:
Most viewed video is 9 Billion and is displayed properly.

Reply:
I am aware of this, but the logic of the function is bad.

Reply:
I'll replace it with a more efficient logic

Reply:
Okay, perhaps look at our human_format method in Imaging.